applications/galaxy.md

```diff
@@ -0,0 +1,205 @@
+# Galaxy: Three-dimensional Web for Polkadot Users
+
+- **Team Name:** Galaxy.Do
+- **Payment Address:** 0xF1dcdA1C9E76BBf32791da01e464A3F55bd7000c (USDC)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 1
+
+## Project Overview :page_facing_up:
+
+### Overview
+
+![image](https://user-images.githubusercontent.com/4569866/228949639-f89bb602-59c5-4b30-917e-8a8b3cdebc20.png)
+
+Web has revolutionized the way we organize and explore knowledge.
+
+Although web pages digitalized physical documents, they also inherited some of limitations.
+
+That's the reason most web pages are vertically scrollable, in one dimension only, imitating physical predecessor.
+
+Whiteboards expanding it's space into second dimension leverage human ability of spatial awareness to enhance the understanding.
+
+As hyper text links navigate user in between of web site pages, same way whiteboards can be also inter-linked to expand information into a third dimension.
+
+Then multiple related whiteboards can be rendered as layers on the same canvas with seamless transition in between the layers, thus expanding the knowledge into a third dimension.
+
+That's the quintessence of Stretch Text concept, which makes it possible to express interconnectedness of all knowledge (intertwingularity)
+
+**Current grant is offering to develop canvas-based browser which allows polkadot users to create, organize, and share knowledge leveraging three-dimensional Stretch Text space.**
```

Comment:
Can you explain what Stretch Text space is? What kind of knowledge will be shared? Is it similar to a forum?

Reply:
> * Can you refer to our [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) in order to fix the top so that the payment address and currency is included?

[force-pushed](https://github.com/w3f/Grants-Program/compare/0d09d2f3cc3afab71dd02270ad1ed28c5be9e018..14d3d55e1cf2ebda6ac04f41cab1093a26956d7f)

> * We typically ask for wireframes or mockups for front-end focused milestones. Do you have any you can provide?

It's based on *[Layer UI](https://github.com/excalidraw/excalidraw/blob/master/src/components/LayerUI.tsx)

![image](https://user-images.githubusercontent.com/4569866/228989761-9b3405a8-a098-429c-a513-cc158733a80e.png)
At the top center you have an UI island with button tools, and at the top right an island with connect wallet button.

* Mentioning "Layer" UI, I must clarify it's not related to the concept of "Layer Links" described in application.

> * Can you please add more technical specifics into the deliverables, such as the tech stack you will be using and what languages? For example, will the smart contract be written in ink!?

That's right, smart contract will be written in ink!
As mentioned, frontend in React, and backend in Deno/Rust stack.

> * Is the Galaxy browser only meant to be for Polkadot/Kusama users? Or users of all Dapps?

Anyone can open galaxy links in read-only mode, but in order to save a new document/layer, user have to sign a transaction in Kusama network.

> Can you explain what Stretch Text space is? What kind of knowledge will be shared? Is it similar to a forum?

Imagine it's a like a forum where next to every message you can see all the relevant information about the concepts mentioned in the message. Hopefully that makes sense.

Layer Link Example: https://galaxy.do/maps/?name=books